specification/app/resource-manager/Microsoft.App/preview/2024-10-02-preview/ContainerAppsLabelHistory.json

```diff
@@ -0,0 +1,225 @@
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ResourceGroupNameParameter"
+          },
+          {
+            "name": "containerAppName",
+            "in": "path",
+            "description": "Name of the Container App.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "name": "labelName",
+            "in": "path",
+            "description": "Name of the Container App label.",
+            "required": true,
+            "type": "string",
+            "pattern": "^[-\\w\\._\\(\\)]+$"
+          },
+          {
+            "$ref": "../../../../../common-types/resource-management/v3/types.json#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK",
+            "schema": {
+              "$ref": "#/definitions/LabelHistory"
+            }
+          },
+          "default": {
+            "description": "Common error response.",
+            "schema": {
+              "$ref": "./CommonDefinitions.json#/definitions/DefaultErrorResponse"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "Get Container App's single label history": {
+            "$ref": "./examples/LabelHistory_Get.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "LabelHistory": {
+      "description": "Container App Label History.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "../../../../../common-types/resource-management/v5/types.json#/definitions/ProxyResource"
+        }
+      ],
+      "properties": {
+        "properties": {
+          "description": "Container App Label History resource specific properties",
+          "type": "object",
+          "properties": {
+            "records": {
+              "description": "List of label history records.",
+              "type": "array",
+              "readOnly": true,
+              "items": {
+                "$ref": "#/definitions/LabelHistoryRecordItem"
+              },
+              "x-ms-identifiers": [
+                "label"
+              ]
+            }
+          }
+        }
+      },
+      "x-ms-client-flatten": true
+    },
+    "LabelHistoryRecordItem": {
+      "description": "Container App Label History Item resource specific properties",
+      "type": "object",
+      "properties": {
+        "revision": {
+          "description": "Container App revision name that label was applied to.",
+          "type": "string",
+          "readOnly": true
+        },
+        "start": {
+          "format": "date-time",
+          "description": "Timestamp describing when the label was applied to the revision.",
+          "type": "string",
```

Comment:
Should it be DateTime?

Reply:
I think date-time is correct https://swagger.io/docs/specification/v3_0/data-models/data-types/

i think DateTime a datatype used for tables